### ISSUE TITLE
Zooming in too much breaks plots

### ISSUE THREAD

**Issue by shobart** [nasa/openmctweb#440]
Shown to Charles & Pete in today's testathon.

If you zoom in too much on a plot, the marquee box disappears and  the axis disappears.

To reproduce:

1. open a new session
2. navigate to My Items -> Battery Layout
3. Zoom in the plot BMS_VOLTAGE -- continue zooming smaller and smaller until you see the problem.  This will take maybe 10-12 zooms.

**Comment by charlesh88** [comment]
Is this still a problem with new plot functionality on the way?

**Comment by larkin** [comment]
Yes, can re-evaluate after #1557

**Comment by davetsay** [comment]
@charlesh88 , i wonder if this issue was what you were seeing when testing time conductor and zooming resulted in invalidZ start and end bounds.

**Comment by shefalijoshi** [comment]
Is this a duplicate of #7926 ?

**Comment by unlikelyzero** [comment]
Still reproduces on master (3aaa0f05e). Opening a Sine Wave Generator's plot and repeatedly clicking zoom-in (~25x) degenerates the Y-axis: all tick labels collapse to 0/-0 with no meaningful scale, once the range shrinks below float precision. No console error on the Vue plot. Root cause looks like a missing minimum-zoom / range-floor guard. (Possible duplicate of #7926 — worth cross-checking.)
